layout/plan.h

```diff
@@ -0,0 +1,428 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2016-2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ * Original author: Nikita Danilov <nikita.danilov@seagate.com>
+ * Original creation date: 07-Jun-2016
+ */
+
+#pragma once
+
+#ifndef __MOTR_LAYOUT_PLAN_H__
+#define __MOTR_LAYOUT_PLAN_H__
+
+struct m0_layout {
+	struct m0_entity la_ent;
+};
```

Comment:
m0_layout is already defined in "layout/layout.h", and please note, the two definitions are different.

Can we just have a forward declaration here?

Reply:
I know this file has not been included in the Makefile. But removing these obvious bug seems necessary.

Reply:
@huanghua78, I asked @nikitadanilov above about these structs, let's hear from him. Of course, we shouldn't land it as it is.